docs/apache-airflow-providers-cncf-kubernetes/operators.rst

```diff
@@ -115,14 +905,169 @@ alongside the Pod. The Pod must write the XCom value into this location at the `
 
 See the following example on how this occurs:
 
-.. exampleinclude:: /../../airflow/providers/cncf/kubernetes/example_dags/example_kubernetes.py
-    :language: python
-    :start-after: [START howto_operator_k8s_write_xcom]
-    :end-before: [END howto_operator_k8s_write_xcom]
 
-Reference
-^^^^^^^^^
-For further information, look at:
+- Example Dag : KubernetesPodOperator task writes contents to be returned to ``/airflow/xcom/return.json`` and reading
+  values returned using ``xcom_pull(key, task_ids)``.
+
+.. code-block:: python
```

Comment:
Can you please move these code segments into example_dag files or some other external file and pull in using tags? That way we can re-use these code snippets later and have python linting when editing them.

Reply:
I have extracted example and tagged in doc.